app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrap_cache(*args):
+        stored_result = cache_dict.get(args, False)
```

Comment:
You don't need this variable. You use it only in `if` statement but you can simply use `cache_dict[args]`